modules/default/compliments/compliments.js

```diff
@@ -1,4 +1,8 @@
 /* global Cron */
+/* global global */
+
+console.log("window name='"+window.name+"'")
+const compliments_test_mode=(window.name=='jsdom')?true:false
```

Comment:
defined but not used

Reply:
the wanted testcase to run
you see my two approaches, both fail